sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslSqlStdOperatorsTest.java

```diff
@@ -557,6 +566,326 @@ public void testStringFunctions() throws Exception {
     pipeline.run();
   }
 
+  @Test
+  @SqlOperatorTest(name = "ABS", kind = "OTHER_FUNCTION")
+  public void testAbs() {
+    ExpressionChecker checker =
+        new ExpressionChecker()
+            .addExpr("ABS(c_integer)", Math.abs(INTEGER_VALUE))
+            .addExpr("ABS(c_bigint)", Math.abs(LONG_VALUE))
+            .addExpr("ABS(c_smallint)", (short) Math.abs(SHORT_VALUE))
+            .addExpr("ABS(c_tinyint)", (byte) Math.abs(BYTE_VALUE))
+            .addExpr("ABS(c_double)", Math.abs(DOUBLE_VALUE))
+            .addExpr("ABS(c_float)", Math.abs(FLOAT_VALUE))
+            .addExpr("ABS(c_decimal)", new BigDecimal(Math.abs(ONE.doubleValue())));
+    checker.buildRunAndCheck();
+  }
+
+  @Test
+  @SqlOperatorTest(name = "LN", kind = "OTHER_FUNCTION")
+  public void testLn() {
+    ExpressionChecker checker =
+        new ExpressionChecker()
+            .addExpr("LN(c_integer)", Math.log(INTEGER_VALUE))
+            .addExpr("LN(c_bigint)", Math.log(LONG_VALUE))
+            .addExpr("LN(c_smallint)", Math.log(SHORT_VALUE))
+            .addExpr("LN(c_tinyint)", Math.log(BYTE_VALUE))
+            .addExpr("LN(c_double)", Math.log(DOUBLE_VALUE))
+            .addExpr("LN(c_float)", Math.log(FLOAT_VALUE))
+            .addExpr("LN(c_decimal)", Math.log(ONE.doubleValue()));
+    checker.buildRunAndCheck();
+  }
+
+  @Test
+  @SqlOperatorTest(name = "SQRT", kind = "OTHER_FUNCTION")
+  public void testSqrt() {
+    ExpressionChecker checker =
+        new ExpressionChecker()
+            .addExpr("SQRT(c_integer)", Math.sqrt(INTEGER_VALUE))
+            .addExpr("SQRT(c_bigint)", Math.sqrt(LONG_VALUE))
+            .addExpr("SQRT(c_smallint)", Math.sqrt(SHORT_VALUE))
+            .addExpr("SQRT(c_tinyint)", Math.sqrt(BYTE_VALUE))
+            .addExpr("SQRT(c_double)", Math.sqrt(DOUBLE_VALUE))
+            .addExpr("SQRT(c_float)", Math.sqrt(FLOAT_VALUE))
+            .addExpr("SQRT(c_decimal)", Math.sqrt(ONE.doubleValue()));
+    checker.buildRunAndCheck();
+  }
+
+  @Test
+  @SqlOperatorTest(name = "ROUND", kind = "OTHER_FUNCTION")
```

Comment:
Can we have more granular `kind`? E.g. `"BASIC_MATH"` for things like `max, min, sqrt..`, `"TRIGONOMETRY"` for `sin, cos..`, `"RANDOM"` for randoms

Reply:
`kind` comes from Calcite. It is not defined by us. See: `https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/sql/SqlKind.java`

Reply:
Ah, didn't realize it